packages/daimo-api/src/network/bundlerClient.ts

```diff
@@ -21,12 +43,183 @@ interface GasPrice {
+        "0000000000000000000000000000000000000000000000000000000000000020",
+        "0000000000000000000000000000000000000000000000000000000000000001",
+        "0000000000000000000000000000000000000000000000000000000000000020",
+        "000000000000000000000000" + this.inflatorCoinAddr.slice(2),
+        "0000000000000000000000000000000000000000000000000000000000000000",
+        "0000000000000000000000000000000000000000000000000000000000000060",
+        "0000000000000000000000000000000000000000000000000000000000000044",
+        "a9059cbb", // transfer
+        "000000000000000000000000(.{40})",
+        "0000000000000000000000000000000000000000000000000000(.{12})",
+        "00000000000000000000000000000000000000000000000000000000",
+        "$",
+      ].join(""),
+      "i"
+    );
+    m = calldataRegex.exec(op.callData);
+    if (!m) throw new Error("can't compress, bad callData: " + op.callData);
+    ret.push(`0x${m[1]}`); // to
+    ret.push(`0x${m[2]}`); // amount
+
+    // op signature
+    const sigRegex = new RegExp(
+      [
+        "^",
+        "0x(.{16})", // sig version, validUntil, keySlot
+        "0000000000000000000000000000000000000000000000000000000000000020",
+        "00000000000000000000000000000000000000000000000000000000000000c0",
+        "0000000000000000000000000000000000000000000000000000000000000120",
+        "0000000000000000000000000000000000000000000000000000000000000017",
+        "0000000000000000000000000000000000000000000000000000000000000001",
+        "(.{64})", // sig r
+        "(.{64})", // sig s
+        "0000000000000000000000000000000000000000000000000000000000000025",
+        "0000000000000000000000000000000000000000000000000000000000000000",
+        "0500000000000000000000000000000000000000000000000000000000000000",
+        "000000000000000000000000000000000000000000000000000000000000005a",
+        stringToHex('{"type":"webauthn.get","challenge":"').slice(2),
+        "(.{104})", // authenticator challenge
+        stringToHex('"}').slice(2),
+        "000000000000",
+        "$",
+      ].join(""),
+      "i"
+    );
+    m = sigRegex.exec(op.signature);
+    if (!m) throw new Error("can't compress, bad signature: " + op.signature);
+    ret.push(`0x${m[1]}`); // sig version, validUntil, keySlot
+    ret.push(`0x${m[2]}`); // sig r
+    ret.push(`0x${m[3]}`); // sig s
+    ret.push(`0x${m[4]}`); // authenticator challenge
+
+    // paymaster signature, if present
+    const paymasterAddr = this.inflatorPaymaster;
+    const paymasterRegex = new RegExp(`${paymasterAddr}(.*)$`, "i");
+    m = paymasterRegex.exec(op.paymasterAndData);
+    if (!m) throw new Error("can't compress, bad paym.:" + op.paymasterAndData);
+    ret.push(`0x${m[1]}`); // paymaster data
+
+    return concatHex(ret);
+  }
+
+  async sendCompressedOpToBulk(
+    compressed: Hex,
+    walletClient: WalletClient<Transport, Chain, Account>
+  ) {
+    const txHash = await walletClient.writeContract({
```

Comment:
shouldn't this go through the `ViemClient` instead (to prevent nonce reuse as in deploys?)